runtime/cfdumper/main.c

```diff
@@ -2330,7 +2330,7 @@ static I_32 processClassFile(J9CfrClassFile* classfile, U_32 dataLength, char* r
 			break;
 
 		case ACTION_structure:
-			dumpClassFile(classfile);
+			dumpClassFile(classfile, flags);
```

Comment:
These flags don't appear to be flags that contain BCT_MajorClassFileVersionMask. I traced them back to the buildFlags() function.

Reply:
Its not clear to me if the buildFlags() contains the class version, however these same flags are also passed into `cfreader.c:j9bcutil_readClassFileBytes`and used to determine the vm version.